File: Daily Challanges/2026-03-March/2026-03-28.py
def pascal_row(n):
    row = [1]
    for _ in range(n - 1):
        new_row = [1]
        for j in range(len(row) - 1):
            new_row.append(row[j] + row[j + 1])
        new_row.append(1)
        row = new_row
    return row

print(pascal_row(1))
print(pascal_row(2))
print(pascal_row(3))
print(pascal_row(5))

File: Daily Challanges/2026-03-March/2026-03-28.js
function pascalRow(n) {
  let row = [1];
  for (let i = 0; i < n - 1; i++) {
    let newRow = [1];
    for (let j = 0; j < row.length - 1; j++) {
      newRow.push(row[j] + row[j + 1]);
    }
    newRow.push(1);
    row = newRow;
  }
  return row;
}

console.log(pascalRow(1));
console.log(pascalRow(2));
console.log(pascalRow(3));
console.log(pascalRow(5));
